File: Application Java/Planning/src/metier/Personne.java
package metier;

import java.util.Date;

public class Personne {
     
    private String nom;
    private String prenom;
    private String nationnalite;
    private int personneID;
    private int age;
    private Date dateNaissance;
    
    public Personne(String newNom, String newPrenom, String newNationnalite, int newPersonneID, int newAge, Date newDateNaissance) {
         this.nom = newNom;
         this.prenom = newPrenom;
         this.nationnalite = newNationnalite;
         this.personneID = newPersonneID;
         this.age = newAge;
         this.dateNaissance = newDateNaissance;
    }
    
    public String getNom() {
        return this.nom;
    }
    
    public void setNom(String newNom) {
        this.nom = newNom;
    }
    
    public String getPrenom() {
        return this.prenom;
    }
    
    public void setPrenom(String newPrenom) {
        this.prenom = newPrenom;
    }
    
    public String getNationnalite() {
        return this.nationnalite;
    }
    
    public void setNationnalite(String newNationnalite) {
        this.nationnalite = newNationnalite;
    }
    
    public int getPersonneID() {
        return this.personneID;
    }
    
    public void setPersonneID(int newPersonneID) {
        this.personneID = newPersonneID;
    }
    
    public int getAge() {
        return this.age;
    }
    
    public void setAge(int newAge) {
        this.age = newAge;
    }
    
    public Date getDateNaissance() {
        return this.dateNaissance;
    }
    
    public void setDateNaissance(Date newDateNaissance) {
        this.dateNaissance = newDateNaissance;
    }

}

File: Application Web/entities/Personne.php
<?php

class Personne {
	
	private $_nom;
	private $_prenom;
	private $_nationnalite;
	private $_personneID;
	private $_age;
	private $_dateNaissance;
	private $_metier;
	
	public function __construct($nom, $prenom, $nationnalite, $personneID, $age, $dateNaissance, $metier) {
		$this -> _nom = $nom;
		$this -> _prenom = $prenom;
		$this -> _nationnalite = $nationnalite;
		$this -> _personneID = $personneID;
		$this -> _age = $age;
		$this -> _dateNaissance = $dateNaissance;
		$this -> _metier = $metier;
	}
	
	public function getNom() {
        return $this -> _nom;
    }
    
    public function setNom($nom) {
        $this -> _nom = $nom;
    }
	
	public function getPrenom() {
        return $this -> _prenom;
    }
    
    public function setPrenom($prenom) {
        $this -> _prenom = $prenom;
    }
	
	public function getNationnalite() {
        return $this -> _nationnalite;
    }
    
    public function setNationnalite($nationnalite) {
        $this -> _nationnalite = $nationnalite;
    }
	
	public function getPersonneID() {
        return $this -> _personneID;
    }
    
    public function setPersonneID($personneID) {
        $this -> _personneID = $personneID;
    }
	
	public function getAge() {
        return $this -> _age;
    }
    
    public function setAge($age) {
        $this -> _age = $age;
    }
	
	public function getDateNaissance() {
        return $this -> _dateNaissance;
    }
    
    public function setDateNaissance($dateNaissance) {
        $this -> _dateNaissance = $dateNaissance;
    }
	
	public function getMetier() {
        return $this -> _metier;
    }
    
    public function setMetier($metier) {
        $this -> _metier = $metier;
    }
	
}

?>
